pkg/localize/locales/en/cmd/kafka_topic_list.en.toml

```diff
@@ -19,4 +19,16 @@ $ rhoas kafka topic list -o json
 '''
 
 [kafka.topic.list.log.info.noTopics]
-one = 'Kafka instance "{{.InstanceName}}" has no topics. Run "rhoas kafka topic create <topic-name>" to create one'
\ No newline at end of file
+one = 'Kafka instance "{{.InstanceName}}" has no topics. Run "rhoas kafka topic create <topic-name>" to create one'
```

Comment:
Probably this message should not be used when the search flag is use - there could be topics but not showing up because of the filter value.

Reply:
Indeed! `--search` for kafka instances returns `No kafka instances were found` for both no instance created and no instance found after searching. Should we modify the message in same manner for consistency?

Reply:
`No Kafka instance were found` is good because it does not call out that there are no topics - just that we did not find any. Sure, let's change the topic message to be consistent :)